lib/open_project/translations/models/locales_updater_configuration.rb

```diff
@@ -1,14 +1,45 @@
 require 'yaml'
 
 class LocalesUpdaterConfiguration
-  CONFIGURATION_FILE = ENV['CONFIGURATION_FILE'] || begin
-    # root of OpenProject-Translation
-    configuration_file = Pathname(__FILE__) + '../../../../../'
-    configuration_file + 'configuration.yml'
+  REQUIRED_KEYS = [:uri, :api_key, :project_id, :version,
+                   :branch]
+
+  def self.path_to_configuration_file
+    ENV['CONFIGURATION_FILE'] || begin
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.